web/src/OverviewTableTriggerButton.tsx

```diff
@@ -46,63 +41,3 @@ export let TriggerButtonRoot = styled(InstrumentedButton)`
     fill: ${Color.blue};
   }
 `
-
-type TriggerButtonProps = {
-  isBuilding: boolean
-  hasBuilt: boolean
-  triggerMode: TriggerMode
-  hasPendingChanges: boolean
-  isQueued: boolean
-  resourceName: string
-  analyticsTags: Tags
-  onTrigger: () => void
-}
-
-function OverviewTableTriggerButton(props: TriggerButtonProps) {
-  let isManual =
-    props.triggerMode === TriggerMode.TriggerModeManual ||
-    props.triggerMode === TriggerMode.TriggerModeManualWithAutoInit
-  let isAutoInit =
-    props.triggerMode === TriggerMode.TriggerModeAuto ||
-    props.triggerMode === TriggerMode.TriggerModeManualWithAutoInit
-
-  // clickable (i.e. trigger button will appear) if it doesn't already have some kind of pending / active build
-  let clickable =
-    !props.isQueued && // already queued for manual run
-    !props.isBuilding && // currently building
-    !(isAutoInit && !props.hasBuilt) // waiting to perform its initial build
-
-  let isEmphasized = false
-  if (clickable) {
-    if (props.hasPendingChanges && isManual) {
-      isEmphasized = true
-    } else if (!props.hasBuilt && !isAutoInit) {
-      isEmphasized = true
-    }
-  }
-
-  let classes = []
-  if (!clickable) {
-    classes.push("is-disabled")
```

Comment:
hmmmm...what about these css classes? looks like they don't exist in the new class, but they're still used in the styled component above

Reply:
fixed, thanks